.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+Metrics/LineLength:
+  Max: 120
+Style/Documentation:
+  Enabled: false
+
```

Comment:
Depending on the results we get when running this, we may need to add some entries to this:
```yaml
Style/PredicateName:
  NameWhitelist:
    - is_a?
```